### DISCUSSION TITLE
Default log output to stdout

### DISCUSSION THREAD

**Discussion by lizrice** [aquasecurity/kube-bench#662] [Development]
kube-bench uses glog and it appears that by default, it expects logs to go to file (unless you specify `--logtostderr`). I wonder if it would be better to default to sending logs to stdout in kube-bench. This would make it much easier for users to get logs (and makes more sense in a kubernetes cluster). Any thoughts?

**Comment by mozillazg** [comment]
+1 for change to default to sending logs to stdout or stderr.
IMHO, sending to stderr instead of stdout is better(same as `--logtostderr`)

**Comment by lizrice** [comment]
Perhaps the simplest approach would be to change our default of `--logtostderr` to be true

**Reply by mozillazg** [reply]
> change our default of --logtostderr to be true

👍

**Reply by lizrice** [reply]
Fixed by #696 - thank you @mozillazg!
